pkgs/development/python-modules/nominal-api-protos/default.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  protobuf,
+}:
+
+buildPythonPackage rec {
+  pname = "nominal-api-protos";
+  version = "0.708.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "nominal_api_protos";
+    hash = "sha256-EYyBRmmCq4OA6xgf4JpajUtlJClkxxPn48Wmmy2mqN4=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ protobuf ];
+
+  pythonImportsCheck = [ "nominal_api_protos" ];
+
+  meta = {
+    description = "Generated protobuf client for the Nominal API";
+    homepage = "https://github.com/nominal-io/nominal-client";
+    maintainers = with lib.maintainers; [ alkasm ];
+    license = lib.licenses.mit;
```

```suggestion
    license = lib.licenses.unfree;
```
Upstream doesn't specify a license.

---

Got it. MIT is ultimately correct here, but since it's not distributed with a license file currently, I can keep this as unfree for now.

---

Why is MIT correct? It would be nice if upstream at least had a public repo.

---

Since the description says "generated", maybe we can reproduce that generation?

---

> It would be nice if upstream at least had a public repo.
> Since the description says "generated", maybe we can reproduce that generation?

We likely will be able to do that in the future, but currently the defs are in a private repo.